profiles/deveshp007.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/72290650/208826623-5a77a4e2-8f50-4d22-8302-6b35f7b420b1.png",
+    "issueId": 1581,
+    "name": "Devesh Pandey",
+    "username": "deveshp007"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```